ClavaLaraApi/src-lara-clava/clava/clava/code/Outliner.js

```diff
@@ -154,7 +153,7 @@ class Outliner {
 
         // actions before the function call
         const type = returnStmts[0].children[0].type;
-        const id = Outliner.#getUniqueVarID();
+        const id = IdGenerator.next();
```

Comment:
Just a detail, if you want to customize the ID, and have a separate numbering for functions and variables, you can pass a string as argument, `IdGenerator.next("var_")` will return `var_0`, `var_1`, etc

Reply:
That's nice, I've updated accordingly.